analyzers/tests/SonarAnalyzer.Test/TestCases/ConditionalStructureSameImplementation_If.cs

```diff
@@ -127,6 +127,45 @@ public void Exception(int a)
                 DoSomething1();
             }
         }
+
+        public bool FalseNegative(string s, out int kind)
+        {
+            kind = 0;
+            if (s.StartsWith("__Iterator", StringComparison.Ordinal))
+            { // Secondary [Following]
+                kind = 42;
+                return true;
+            }
+            else if (s.StartsWith("__async", StringComparison.Ordinal))
+            { // Noncompliant [Following]
+                kind = 42;
+                return true;
+            }
+            else if (s.StartsWith("__AnonStorey", StringComparison.Ordinal))
+            {
+                kind = 21;
+                return true;
+            }
+
+            while (true)
+            {
+                if (s == string.Empty)
+                { // Secondary [FN3]
```

Comment:
Why `FN3` and `TP4`? Apart from these two labels, I only see `Following` above, which doesn't follow the same naming pattern.
Moreover using `FN` may lead the reader believe that there is still a False Negative there, whereas, if I get it correctly, it's now a True Positive, that fixes the False Negative that this PR targets.
I would change label with something else, to avoid such potential confusion.